src/common/Makefile

```diff
@@ -38,6 +38,7 @@ redis:
 hiredis:
```

Comment:
I just tried running

```
@ray.remote
def f():
  return 1
```

And it printed

```
[ERROR] (state/table.c:70: errno: Operation now in progress) Table command with timer ID 0 failed
```

If that is something that happens normally while Ray is functioning correctly, then there it shouldn't be an error or even a warning. It should be INFO or DEBUG, right?